src/components/command-bar/commands/search/components/tab-contents-cta/index.tsx

```diff
@@ -0,0 +1,31 @@
+import { IconArrowRight16 } from '@hashicorp/flight-icons/svg-react/arrow-right-16'
+import Card from 'components/card'
+import Text from 'components/text'
+import StandaloneLink from 'components/standalone-link'
+import s from './tab-contents-cta.module.css'
+
+interface TabContentsCtaProps {
+	href: string
+	icon: JSX.IntrinsicElements['svg']
+	text: string
+}
+
+const TabContentsCta = ({ href, icon, text }: TabContentsCtaProps) => {
+	return (
+		<Card className={s.cta} elevation="base">
```

Comment:
💭 This is maybe more a design slash UX question, but have we considered a `CardLink` here, for consistency in the focus styling and pointer target area of the link?

Because this card is styled in a way where it takes up the same kind of space as a result, personally I find it a little unexpected that the results are these nice large target areas, but then this CTA is only clickable on the `StandaloneLink` part:

https://user-images.githubusercontent.com/4624598/216212841-82c3fcdc-1ffa-4a00-8cb3-8ca61bc49c68.mp4

Reply:
Can you create a ticket for this to be discussed? This is functionality that existed before this PR, so I think it's out of scope to change for this one.

Reply:
Yeah I agree with you @zchsh and think we could improve this. Let's just not improve it now so @ashleemboyer can keep rolling. Thanks for the feedback.

Reply:
Sweet, sounds good! Here's a ticket in case it's helpful: 🎟 [[discuss] integrations search results "Show more" link activation target area](https://app.asana.com/0/1202097197789424/1203885784918902/f)